src/components/MiniCart/MiniCart.js

```diff
@@ -146,9 +153,10 @@ export default class MiniCart extends Component {
   }
 
   render() {
-    const { classes, uiStore } = this.props;
+    const { cart, classes, uiStore } = this.props;
     const { isCartOpen } = uiStore;
-    const id = isCartOpen ? "simple-popper" : null;
+    const id = (isCartOpen) ? "simple-popper" : null;
+    const cartQuantity = (cart && Array.isArray(cart.items) && cart.items.length) || 0;
```

Comment:
Use `cart.totalItemQuantity`

Reply:
That didn't exist. I looked for it but couldn't find it.

Reply:
Hmm that's weird. It definitely wasn't there when I checked before. I had tried both logging and also a debugger. But lo and behold, it's there now.